molgenis/molgenis-emx2
# chore: refactor tests so not have to wait on initDatabase all the time

Should feel 'faster' when doing test driven developing using your Java editor

how to test:
- before, when in Intellij if you tested for example 'rdf' then first code from *sql package would be run
- now it should now start immediately instead of having to wait for some unrelated tests to run first in the *sql package

What is nice, is that the 'sql' tests could now be moved back to the *sql module instead of having seperate *sql-it module.
What is also nice is that you can run 'gradle initDatabase' outside

- [x] DATABASE INITIALIZING should be only once in test output, at the beginning
